plugins/cherwell/unit_test/util.py

```diff
@@ -0,0 +1,92 @@
+from komand_cherwell.connection import Connection
+from komand_cherwell.connection.schema import Input
+from typing import Dict
+import insightconnect_plugin_runtime
+import json
+import logging
+import sys
+import os
+
+sys.path.append(os.path.abspath("../"))
+
+STUB_URL = "http://0.0.0.0"
+STUB_AUTHENTICATION_MODE = "Internal"
+STUB_CLIENT_ID = {"secretKey": "a5zy0a6g-504e-46bz-84xx-1b3f5ci36l99"}
+STUB_USERNAME_AND_PASSWORD = {"username": "user@example.com", "password": "mypassword"}
+STUB_QUERY_PARAMS = "?auth_mode=Internal"
+
+
+class Util:
+    @staticmethod
+    def default_connector(action: insightconnect_plugin_runtime.Action, custom_params: Dict = {}):
+        default_connection = Connection()
+        default_connection.logger = logging.getLogger("connection logger")
+        params = {
+            Input.URL: STUB_URL,
+            Input.CLIENT_ID: STUB_CLIENT_ID,
+            Input.SSL_VERIFY: False,
+            Input.AUTHENTICATION_MODE: STUB_AUTHENTICATION_MODE,
+            Input.USERNAME_AND_PASSWORD: STUB_USERNAME_AND_PASSWORD,
+        }
+        Util.overwrite_params(params, custom_params)
+        default_connection.connect(params)
+        action.connection = default_connection
+        action.logger = logging.getLogger("action logger")
+        return action
+
+    @staticmethod
+    def overwrite_params(dict_one, dict_two):
+        for key, value in dict_two.items():
+            if key in dict_one:
+                dict_one[key] = value
+        return dict_one
```

Comment:
nothing wrong with this (I think it essentially is just writing out what .update() is doing behind the scenes) but you could just do `dict_one.update(dict_two)` or or line 31 do `params.update(custom_params)` ?